client/src/components/MultiJurisdictionAudit/StatusBox/index.tsx

```diff
@@ -367,6 +367,10 @@ export const JurisdictionAdminStatusBox = ({
     const details = [
       `${numCompleted} of ${auditBoards.length} audit boards complete.`,
```

Comment:
thinking about this more, this sentence really only makes sense for online data entry audits, since in all offline audits, all of the audit board results are entered at once

Reply:
@jonahkagan yeah, good point. Which means this should work fine for this, but we could also just filter it out for all offline audits. I'm not seeing that online/offline is available in the status box yet, though. I wonder how much it would take to add that in...

Reply:
it's in the audit settings, so should be easy

Reply:
@jonahkagan okay, let me know what you think of my latest update.